Multisig/Logic/Models/Safe+CoreDataProperties.swift

```diff
@@ -18,6 +18,7 @@ extension Safe {
     @NSManaged public var additionDate: Date?
     @NSManaged public var address: String?
     @NSManaged public var name: String?
+    @NSManaged public var ownersData: Data?
```

Comment:
is there need to do something extra for the lightweight migration 🤔? versioning the xcdatamodel, or something?